.bandit.yml

```diff
@@ -0,0 +1,302 @@
+any_other_function_with_shell_equals_true:
+  no_shell:
+    - os.execl
+    - os.execle
+    - os.execlp
+    - os.execlpe
+    - os.execv
+    - os.execve
+    - os.execvp
+    - os.execvpe
+    - os.spawnl
+    - os.spawnle
+    - os.spawnlp
+    - os.spawnlpe
+    - os.spawnv
+    - os.spawnve
+    - os.spawnvp
+    - os.spawnvpe
+    - os.startfile
+  shell:
+    - os.system
+    - os.popen
+    - os.popen2
+    - os.popen3
+    - os.popen4
+    - popen2.popen2
+    - popen2.popen3
+    - popen2.popen4
+    - popen2.Popen3
+    - popen2.Popen4
+    - commands.getoutput
+    - commands.getstatusoutput
+  subprocess:
+    - subprocess.Popen
+    - subprocess.call
+    - subprocess.check_call
+    - subprocess.check_output
+    - subprocess.run
+assert_used:
+  skips: ['*_test.py', '*test_*.py']
```

Comment:
Added a bandit configuration file to avoid bandit raising errors about `assert` statements in our tests.